### PR TITLE
Fix log messages and var names in integr test

### PR DESCRIPTION
**Pull request by erictune**
It says "pods" where the things are actually "containers".

### CODE REVIEW

File: cmd/integration/integration.go [hunk 1]
```diff
@@ -764,29 +764,29 @@ func main() {
...
 		}
 	}
 	for _, p := range fakeDocker2.Created {
 		// The last 8 characters are random, so slice them off.
 		if n := len(p); n > 8 {
-			createdPods.Insert(p[:n-8])
+			createdConts.Insert(p[:n-8])
 		}
 	}
 	// We expect 9: 2 pod infra containers + 2 pods from the replication controller +
```
Comment (vmarmol) [inline cmd/integration/integration.go]: Should this comment be updated too? 2 containers from the replication controller pods

File: cmd/integration/integration.go [hunk 2]
```diff
@@ -764,29 +764,29 @@ func main() {
...
 	//              1 pod infra container + 2 pods from the URL +
 	//              1 pod infra container + 1 pod from the service test.
-	if len(createdPods) != 9 {
-		glog.Fatalf("Expected 9 pods; got %v\n\nlist of created pods:\n\n%#v\n\nDocker 1 Created:\n\n%#v\n\nDocker 2 Created:\n\n%#v\n\n", len(createdPods), createdPods.List(), fakeDocker1.Created, fakeDocker2.Created)
+	if len(createdConts) != 9 {
+		glog.Fatalf("Expected 9 pods; got %v\n\nlist of created pods:\n\n%#v\n\nDocker 1 Created:\n\n%#v\n\nDocker 2 Created:\n\n%#v\n\n", len(createdConts), createdConts.List(), fakeDocker1.Created, fakeDocker2.Created)
```
Comment (vmarmol) [inline cmd/integration/integration.go]: nit: expected 9 "containers", list of created "containers", etc?

Comment (erictune) [inline cmd/integration/integration.go]: ok

File: cmd/integration/integration.go (hunk 1 above)
Comment (erictune) [inline cmd/integration/integration.go]: ok

**Comment by vmarmol** [comment]
@erictune did you make a new push I didn't see?

**Comment by erictune** [comment]
pushed for reals.

**Comment by vmarmol** [comment]
LGTM. Thanks for fixing the nits :)

**Pull request merged**